From ad526fcec919146653b57889d60e0eec83ad7a0a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 5 Jun 2014 12:30:22 +0200
Subject: [PATCH 7/7] udev-rules: add uaccess rules

plymouth needs the seat rules, also without systemd

Conflicts:
	modules.d/95udev-rules/module-setup.sh
---
 modules.d/95udev-rules/module-setup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/95udev-rules/module-setup.sh b/modules.d/95udev-rules/module-setup.sh
index 2d36712..4e75c0b 100755
--- a/modules.d/95udev-rules/module-setup.sh
+++ b/modules.d/95udev-rules/module-setup.sh
@@ -32,6 +32,8 @@ install() {
         60-pcmcia.rules \
         50-udev.rules 95-late.rules \
         50-firmware.rules \
+        59-scsi-sg3_utils.rules \
+        70-uaccess.rules 71-seat.rules 73-seat-late.rules \
         75-net-description.rules \
         80-net-name-slot.rules 80-net-setup-link.rules \
         "$moddir/59-persistent-storage.rules" \
-- 
2.0.0

